third_party/move/move-model/bytecode/src/lib.rs

```diff
@@ -49,6 +49,9 @@ pub fn print_targets_with_annotations_for_test(
     let mut text = String::new();
     writeln!(&mut text, "============ {} ================", header).unwrap();
     for module_env in env.get_modules() {
+        if !module_env.is_target() {
```

Comment:
prevent panic from `targets.get_targets` below when the module is not the target module